packages/utils/src/crypt/core.ts

```
@@ -46,11 +46,14 @@ if (crypto) {
 }
 
 export class Base {
+  // @ts-ignore
   static create(...args) {
+    // @ts-ignore
     return new this(...args)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)